ci/playbooks/edpm_build_images/run.yml

```diff
@@ -0,0 +1,12 @@
+---
+- hosts: all
+  gather_facts: true
+  tasks:
+    - name: Run EDPM build image
+      ansible.builtin.command:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework"
+        cmd: >-
+          ansible-playbook ci/playbooks/edpm_build_images/edpm_image_builder.yml
+          -e @scenarios/centos-9/base.yml
+          -e @scenarios/centos-9/edpm_build_images_ci.yml
```

Comment:
please add this also:
```
          {%- if cifmw_extras is defined %}
          {%-   for extra_vars in cifmw_extras %}
          -e "{{   extra_vars }}"
          {%-   endfor %}
          {%- endif %}
```
It will help to pass custom scenario files. It is being added in all playbooks [here:](https://github.com/openstack-k8s-operators/ci-framework/pull/277)